### PR TITLE
ci/cd: build and push zot container images

### LINKED ISSUE #188: ci/cd: build and push zot container images to github's container registry (ghcr.io)
Once a release binary is built:
1) upload the binaries along with the release: https://docs.travis-ci.com/user/deployment/releases/
2) upload a container image: https://docs.github.com/en/actions/guides/publishing-docker-images

### PR DESCRIPTION
**Pull request by peusebiu**
to github's container registry, closes #188

### CODE REVIEW

**Comment by rchincha** [comment]
https://docs.github.com/en/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry
^ @peusebiu  are you able to create a PAT at all or not?

**Comment by peusebiu** [comment]
> https://docs.github.com/en/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry
> ^ @peusebiu are you able to create a PAT at all or not?

Yes, I did it and it works for my forked zot.
I understand that secrets.GITHUB_TOKEN is automatically generated for every repo. So I think it should work for anuvu/zot too.

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,16 @@ install:
 script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then travis_wait 60 make; fi
 
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: "DlCyRM85KiUN6p9BvmBjf7gtSHfjbuiJhjIOP19drQfIwxDNvgdQXOV8zSOIS2g8bQK8LOUkUWbVlxRkA5RBVfNb1v8MUaRhByOJ8zl8JMSLs4XcUvcLFENp7Au/6HCWMrPg170f3NvoaJFBCyVFh8nYRys7PcRenRafnvEapTLA6gNMaS6T4PIenwwu/FFfE+jAmW3PHDx+Ln0sgmTpyV+/8MyJrS0X39HUeM9HgX5C0ErUMptlb4LIrN5feGeHSE9agOKT7yIGoa4UzXpm0WaYIhg291S417l8uCS8uCa4j2D7EUJJmYTKi7I56hCJmMqBWttHrQhSRLfe6PKiBEw+ieR8lhcv/B5MMuWO04oypRko1/iDub8rYclzcty7WMqWDp+Fl0wT7SIy9fHL9kE8moIItTQDVxRUV3YaLx2pxD4o8OHea8FFnH/I6AMZkikw0Gk1bZlUmYsH0vFghs45rY71tMoVLgk9WAqeIbcgTgq1XEtdokY+PbnZ01EdlQKLj4LA8Wf1KfCqId4EDycgSF/3TAcWiuPr7cJFV9zgyOI4dzr1lRkLJi9huK4MmTbA5NYsA6Ci/Ca1+1hAUfDwcbSjRTeM4s4sCQnxihzQo66Ba0F18kFGVadkrnh/g1DnsW6NeTyfuukU4JvDg4/4Cyqa0IoG2rdERWgo4Es="
```
Comment (peusebiu) [inline .travis.yml:26]: Will this work with my PAT? for anuvu/zot? it works for my fork.
https://docs.travis-ci.com/user/deployment/releases/#authenticating-with-an-oauth-token

**Comment by peusebiu** [comment]
Ok, this is working on my fork. Do any of you guys know if it will also work on anuvu/zot.
Pushing the image might work because I use the built-in ${{ secrets.GITHUB_TOKEN }}, but I'm not sure about pushing binaries, there I've hardcoded my encrypted PAT.

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**